src/pages/RepoPage/CoverageOnboarding/ActivationBanner/ActivationRequiredSelfHosted/ActivationRequiredSelfHosted.tsx

```diff
@@ -60,8 +61,15 @@ function SeatsAvailable({ isAdmin }: { isAdmin: boolean }) {
   )
 }
 
+interface URLParams {
+  provider: string
+}
+
 function ActivationRequiredSelfHosted() {
-  const { data } = useSelfHostedCurrentUser()
+  const { provider } = useParams<URLParams>()
+  const { data } = useSuspenseQueryV5(
+    SelfHostedCurrentUserQueryOpts({ provider })
```

Comment:
same Q re: provider is case-sensitive here? - https://github.com/codecov/gazebo/pull/3580#discussion_r1895971395

Reply:
Answer in the previous thread

Reply:
Sounds good, thanks for explaining!